Lombiq.VueJs/Assets/Scripts/helpers/rollup-pipeline.js

```diff
@@ -0,0 +1,72 @@
+const all = require('gulp-all');
+const path = require('path');
+const gulp = require('gulp');

```

Comment:
Don't we want to get rid of Gulp? Probably we will soon, and it'll be more challenging to do it later than now. Rollup has a CLI too.

Reply:
Eventually sure, but I don't know if a final decision is made yet. I was under the impression that we are waiting until [OSOE-69](https://lombiq.atlassian.net/browse/OSOE-69) is done. Also I think it should be a separate issue in OSOCE where both app and SFC pipelines are used (as of [OSOCE-64](https://lombiq.atlassian.net/browse/OSOE-64)) to avoid causing problems. 

All this file did was DRY the shared parts between [vue-app-compiler-pipeline.js](https://github.com/Lombiq/Orchard-Vue.js/blob/issue/SNOW-14/Lombiq.VueJs/Assets/Scripts/helpers/vue-app-compiler-pipeline.js) and [vue-sfc-compiler-pipeline.js](https://github.com/Lombiq/Orchard-Vue.js/blob/issue/SNOW-14/Lombiq.VueJs/Assets/Scripts/helpers/vue-sfc-compiler-pipeline.js) that are copied from each other. So it won't make switching any easier or harder.

Reply:
OK, you are right about we don't have a final decision yet. Let's leave it as is now, then.